core/src/result.rs

```diff
@@ -65,6 +66,8 @@ pub enum Error {
         name: StdString,
         message: Option<StdString>,
     },
+    /// Error when restoring a Persistent in a runtime other than the original runtime.
+    DifferentRuntime,
```

Comment:
I think the more proper variant should sounds like `IrrelevantRuntime`.

Reply:
`UnrelatedRuntime` sounds good to me. Pushed changes for that.